pages/ui-components/packages.md

```diff
@@ -123,10 +127,12 @@ Using only package source would result in the following:
 @forward "usa-accordion/src/styles";
 @forward "usa-banner/src/styles";
 @forward "usa-button/src/styles";
+@forward "usa-icon/src/styles";
```

Comment:
usa-accordion/_index.scss had a dependency of usa-icon - assumed we wanted to point to the src/styles here